File: backend/app/models/ObservacaoSala.php
<?php  defined('INITIALIZED') OR exit('You cannot access this file directly');

class ObservacaoSala extends Model {

	private $id;	
	private $descricao;
	private $dataHora;
	private $sala;
	private $usuario;
	private $ativo;

    public function getId()
    {
        return $this->id;
    }

    public function setId($id)
    {
        $this->id = $id;
    }

    public function getDescricao()
    {
        return $this->descricao;
    }

    public function setDescricao($descricao)
    {
        $this->descricao = $descricao;
    }

    public function getDataHora()
    {
        return $this->dataHora;
    }

    public function setDataHora($dataHora)
    {
        $this->dataHora = $dataHora;
    }

    public function getSala()
    {
        return $this->sala;
    }

    public function setSala($sala)
    {
        $this->sala = $sala;
    }

    public function getUsuario()
    {
        return $this->usuario;
    }

    public function setUsuario($usuario)
    {
        $this->usuario = $usuario;
    }

    public function getAtivo()
    {
        return $this->ativo;
    }

    public function setAtivo($ativo)
    {
        $this->ativo = $ativo;
    }
	
	
}

File: Lixo/frontend/src/models/ObservacaoSala.java
<?php

defined('INITIALIZED') OR exit('You cannot access this file directly');

class ObservacaoSala extends Model {

    private $id;
    private $descricao;
    private $dataHora;
    private $idSala;
    private $idUsuario;
    private $ativo;

    function getId() {
        return $this->id;
    }

    function getDescricao() {
        return $this->descricao;
    }

    function getDataHora() {
        return $this->dataHora;
    }

    function getIdSala() {
        return $this->idSala;
    }

    function getIdUsuario() {
        return $this->idUsuario;
    }

    function getAtivo() {
        return $this->ativo;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setDescricao($descricao) {
        $this->descricao = $descricao;
    }

    function setDataHora($dataHora) {
        $this->dataHora = $dataHora;
    }

    function setIdSala($idSala) {
        $this->idSala = $idSala;
    }

    function setIdUsuario($idUsuario) {
        $this->idUsuario = $idUsuario;
    }

    function setAtivo($ativo) {
        $this->ativo = $ativo;
    }

}
